src/backend/api/reviews.js

```diff
@@ -0,0 +1,86 @@
+const express = require("express");
+const reviewsRouter = express.Router();
+const dbcon = require("../database");
```

Comment:
I am not sure why have you chosen `dbcon` instead of `knex` which is easily understandable.

Reply:
thanks again.